libs/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-options/dataset-options.component.ts

```diff
@@ -38,6 +40,9 @@ export class DatasetOptionsComponent implements OnInit {
     this.form.valueChanges.subscribe((value) => {
       this.updateModel(value);
     });
+    this.sharedDataService.showAdvancedOptions$.subscribe((showAdvancedOptions: boolean) => {
+      this.showAdvancedOptions = showAdvancedOptions;
+    });
```

Comment:
this should be done with async pipe. otherwise we need to unsubscribe when component is destroyed.

Reply:
Okay